AdminUi/packages/admin_api_types/lib/src/identities/identity_deletion_process.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:admin_api_types/src/admin_api_types_base.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+part 'identity_deletion_process.g.dart';
+
+@JsonSerializable()
+class IdentityDeletionProcess {
+  final String id;
+  final String status;
+  final DateTime createdAt;
+  final DateTime approvalPeriodEndsAt;
+  final DateTime? approvalReminder1SentAt;
+  final DateTime? approvalReminder2SentAt;
+  final DateTime? approvalReminder3SentAt;
+  final DateTime? approvedAt;
+  final String? approvedByDevice;
+  final DateTime? gracePeriodEndsAt;
+  final DateTime? gracePeriodReminder1SentAt;
+  final DateTime? gracePeriodReminder2SentAt;
+  final DateTime? gracePeriodReminder3SentAt;
+
+  IdentityDeletionProcess({
+    required this.id,
+    required this.status,
+    required this.createdAt,
+    required this.approvalPeriodEndsAt,
+    this.approvalReminder1SentAt,
+    this.approvalReminder2SentAt,
+    this.approvalReminder3SentAt,
+    this.approvedAt,
+    this.approvedByDevice,
+    this.gracePeriodEndsAt,
+    this.gracePeriodReminder1SentAt,
+    this.gracePeriodReminder2SentAt,
+    this.gracePeriodReminder3SentAt,
+  });
+
+  factory IdentityDeletionProcess.fromJson(dynamic json) => _$IdentityDeletionProcessFromJson(json as Map<String, dynamic>);
+  Map<String, dynamic> toJson() => _$IdentityDeletionProcessToJson(this);
+}
+
+@JsonSerializable()
+class IdentityDeletionProcessDetail {
```

Comment:
the detail has less details than the actual deletion process?

@tnotheis is this correct?

Reply:
It is not it has the same, but other details information are not used. I will add them to the type.

Reply:
That's sad but true. ;) I already mentioned that last Monday in our meeting with Norbert.

Reply:
Why cant you change that @tnotheis?

Reply:
We could. But that's gonna be part of a separate PR.

Reply:
But the goal of the PR is to make the flutter Admin UI contain everything the angular one contained before. And the angular one did show exactly what we have here now.
So I would consider "showing more details" are separate feature.

Reply:
True, but I'd like both types to extend each other, we have to do this later then.